examples/throw/throw.bal

```diff
@@ -18,8 +18,8 @@ function readRecord (Record|() value) {
         }
     }
 }
-
-function main (string... args) {
+// Catch a thrown error. 
+function main (string[] args) {
```

Comment:
Above syntax got changed. Can we fix it please?

Reply:
Fixed with new commit.